index.html

```diff
@@ -17,8 +17,13 @@
             <h1>TO DO LIST</h1>
 
             <section class="form">
-                <input type="text" placeholder="Wprowadź zadanie" id='addTaskInput'>
-                <button type="submit" id="addTaskBtn">Dodaj</button>
+                <div class="row">
+                    <input type="text" placeholder="Wprowadź zadanie" class="inputArea" id='addTaskInput'>
```

Comment:
Raz uzywasz `" "` a raz `' '`, wybierz jedno. W htmlowych tagach zazwyczaj jest to jednak `" "`.